tools/vfscount

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+#
+# vfscount	Count some VFS calls.
+#		For Linux, uses BCC, eBPF. See .c file.
+#
+# Written as a basic example of counting functions.
+#
+# Copyright (c) 2015 Brendan Gregg.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 14-Aug-2015	Brendan Gregg	Created this.
+
+from bpf import BPF
+from ctypes import c_ushort, c_int, c_ulonglong
+from time import sleep, strftime
+from sys import stderr
+
+# kernel symbol translation
+ksym_addrs = []			# addresses for binary search
+ksym_names = []			# same index as ksym_addrs
+def load_kallsyms():
+	symfile = "/proc/kallsyms"
+	try:
+		syms = open(symfile, "r")
+	except:
+		print >> stderr, "ERROR: reading " + symfile
+		exit()
+	line = syms.readline()
+	for line in iter(syms):
+		cols = line.split()
+		name = cols[2]
+		if name[:4] != "vfs_":	# perf optimization
+			continue
+		addr = int(cols[0], 16)
+		ksym_addrs.append(addr)
+		ksym_names.append(name)
+	syms.close()
+def ksym(addr):
+	start = -1
+	end = len(ksym_addrs)
+	while end != start + 1:
+		mid = (start + end) / 2
```

Comment:
I get a traceback when mid is used as an index in the line below, please change to:
mid = int((start + end) / 2)